jax/_src/numpy/fft.py

```diff
@@ -553,15 +553,158 @@ def _fft_core_2d(func_name: str, fft_type: xla_client.FftType, a: ArrayLike,
   return _fft_core(func_name, fft_type, a, s, axes, norm)
 
 
-@implements(np.fft.fft2)
 def fft2(a: ArrayLike, s: Shape | None = None, axes: Sequence[int] = (-2,-1),
          norm: str | None = None) -> Array:
+  """Compute a two-dimensional discrete Fourier transform along given axes.
+
+  JAX implementation of :func:`numpy.fft.fft2`.
+
+  Args:
+    a: input array. Must have ``a.ndim >= 2``.
+    s: optional length-2 sequence of integers. Specifies the size of the output
+      along each specified axis. ``s`` should have same length as that of ``axes``.
```

Comment:
We've already said ``s`` needs to be length-2, so no need to mention the length of ``axes``